build-bench-env.sh

```diff
@@ -726,6 +726,12 @@ if test "$setup_lean" = "1"; then
   echo "make -j$procs"
   make -j $procs
   rm -rf ./tests/  # we don't need tests
+  if test ! -d "$devdir/mathlib"; then
+    mkdir "$devdir/mathlib"
+  fi
+  if test ! -f "$devdir/mathlib/leanpkg.toml"; then
```

Comment:
This condition should always be true, no?

Reply:
not if the file is already there... in the spirit of the above change, I opted for `cp -u`, though.